src/Integration.Vsix.UnitTests/CFamily/CFamilyHelperTests.cs

```diff
@@ -101,12 +122,11 @@ public void CreateRequest_ErrorInFileConfigTryGet_IsHandled()
             var rulesConfigProviderMock = new Mock<ICFamilyRulesConfigProvider>();
 
             // Act
-            var request = CFamilyHelper.CreateRequest(loggerMock.Object, projectItemMock.Object, "c:\\dummy\\file.cpp",
+            var request = CFamilyHelper.CreateRequest(logger, projectItemMock.Object, "c:\\dummy\\file.cpp",
                 rulesConfigProviderMock.Object, null);
 
             // Assert
-            AssertPartialMessageLogged(loggerMock,
-                "Unable to collect C/C++ configuration for c:\\dummy\\file.cpp: ");
+            logger.AssertPartialOutputStringExists("c:\\dummy\\file.cpp", "xxx", nameof(InvalidOperationException));
```

Comment:
I added a new test that checks multiple partial strings in a single message.
The `TestLogger` has a method to do that already so I switched to using `TestLogger` consistently in all tests that check message string contents.